code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -687,8 +687,8 @@
 	required_candidates = 4
 	weight = 3
 	cost = 35
-	requirements = list(101,101,101,101,101,101,101,101,101,101)
-	high_population_requirement = 101
+	requirements = list(2,3,3,4,4,4,4,4,4,4)
```

Comment:
If I am not mistaken I believe this is the required cost for each population level (The other ones seem to be in numbers like 60,50,40,30 rather than 2,3,4,5 etc.)

Reply:
Should have checked before I changed it, fixed now